lms/djangoapps/instructor/enrollment.py

```diff
@@ -317,7 +317,11 @@ def _fire_score_changed_for_block(course_id, student, block, module_state_key):
             field_data_cache=cache,
             course_key=course_id
         )
-        points_earned, points_possible = weighted_score(0, module.max_score(), getattr(module, 'weight', None))
+        max_score = module.max_score()
+        if max_score is None:
```

Comment:
That makes sense philosophically. I'm just having a hard time wrapping my head around what would happen here for the learner whose grade currently can't be reset. 

Assuming we don't send the signal, we would catch this error here and the learner's state for the problem would be deleted, allowing her to re-take the exam, but I think she would still see her old grade on the progress page, right?